expanderr.go

```diff
@@ -672,12 +738,12 @@ func logic(w io.Writer, buildctx *build.Context, posn string) error {
 }
 
 var (
-	wFlag      = flag.String("w", "", "write")
-	formatFlag = flag.String("format", "", "output format (source, json). defaults to 'source'")
+	wFlag         = flag.String("w", "", "write")
+	formatFlag    = flag.String("format", "", "output format (source, json). defaults to 'source'")
+	cpuprofile    = flag.String("cpuprofile", "", "write cpu profile `file`")
+	noErrTemplate = flag.String("no-error-template", "", "template to be used if there is no error return value. defaults to a panic")
```

Comment:
In the documentation, let’s add an example, and mention that the syntax is Go’s text/template package.